dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -598,7 +598,7 @@ private DataException excJNDINameNotFound(ComponentMetaData metadata,
      *         as the value. Module and component might be null or might not be
      *         present at all.
      */
-    private String getMetadataIdentifier() {
+    public String getMetadataIdentifier() {
```

Comment:
Let's leave this private for now given that it is only used outside the class by the commented out code.  I think I know a more direct way to get this information that will allow us to group the FutureEMBuilders by module earlier on at the time they are created, and then we will be able to more precisely target them to the most appropriate event for initializing.
```suggestion
    private String getMetadataIdentifier() {
```

Reply:
That's a good point, fixing this now since the build should finish by tomorrow morning